addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiSensorSwitchHandler.java

```diff
@@ -19,14 +20,15 @@
  */
 public class XiaomiSensorSwitchHandler extends XiaomiSensorBaseHandler {
 
-    public XiaomiSensorSwitchHandler(Thing thing) {
-        super(thing);
-    }
+	public XiaomiSensorSwitchHandler(Thing thing) {
```

Comment:
make sure to use the normal formatter - you introduced tabs instead of spaces here